src/features/project/Project.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Project } from '../../entities/Project';
+import { AddProjectInput, ProjectWithID } from './Types';
+import { ProjectDao } from '../../dbutils/project.dao';
+import { MemberDao } from '../../dbutils/member.dao';
+import { ProjectUtils } from './Project.utils';
+
+export class ProjectService {
+  private projectDao = new ProjectDao();
+  private memberDao = new MemberDao();
+  private projectUtils = new ProjectUtils();
+
+  async addProject(input: AddProjectInput): Promise<ProjectWithID> {
+    const {
+      name,
+      description,
+      startDate,
+      endDate,
+      goalAmount,
+      fundRaised,
+      aboutTheCause,
+      planOfAction,
+      file,
+    } = input;
+    const project = new Project();
+    project.name = name;
+    project.description = description;
+    project.startDate = startDate;
+    project.endDate = endDate;
+    project.aboutTheCause = aboutTheCause;
+    project.goalAmount = goalAmount;
+    project.fundRaised = fundRaised;
+    project.planOfAction = planOfAction;
+    project.flyer = file;
```

Comment:
NIT: This is good, but we can this in a single line

`const project = Object.assign(new Project(), input);`